docs/reference/db/configuration.md

```diff
@@ -320,6 +320,8 @@ An optional object that defines the plugins loaded by Platformatic DB.
   or an array of objects composed as follows,
   - `path` (`string`): Relative path to plugin's entry point.
   - `options` (`object`): Optional plugin options.
+  - `encapsulate` (`boolean`): if the path is a folder, it instruct Platformatic to not encapsulate those plugins.
```

Comment:
This sentence doesn't tell me what `true` does vs. what `false` does. It's also not clear what encapsulation actually does here. Maybe link to `@fastify/autoload`?

Reply:
I've linked to the encapsulation docs in Fastify.